pkg/api/mlflow/service/metric/validators.go

```diff
@@ -6,7 +6,18 @@ import (
 )
 
 const (
-	MaxResultsPerPage = 1000000000
+	MaxResultsPerPage                    = 1000000000
```

Comment:
```suggestion
	MaxResultsForMetricsHistoriesRequest                    = 1000000000
```

This is not a pageable endpoint

Reply:
ook. done.